[IMP] sales: correct amazon invoicing and payment registration info (#5142)
odoo/documentation: content/applications/sales/sales/amazon_connector/manage.rst

Issue invoices and register payments
====================================

Issue invoices
--------------

Sending invoices to Amazon customers directly from Odoo is not feasible due to Amazon's policy of
not sharing customer email addresses. Instead, you can manually upload the
invoices generated on Odoo to the Amazon backend.

In addition, it is currently required to manually retrieve VAT numbers from the Amazon backend

> before creating an invoice due to an issue with the Selling Partner API.

Reviewer: This does not sounds very clear for a regular user. I don't think they are aware of what is the "Selling Partner API". 

Maybe something like:
"For your B2B clients, you will have to find their VAT number in Amazon before creating the invoice in Odoo."

> In addition, it is currently required to manually retrieve VAT numbers from the Amazon backend
> before creating an invoice due to an issue with the Selling Partner API.

Reviewer 2: ```suggestion
In addition, it is currently required to manually retrieve VAT numbers from the Amazon backend
before creating an invoice, due to an issue with the Selling Partner API.
```

.. note::
   For :doc:`TaxCloud <../../../finance/accounting/taxes/taxcloud>` users: invoices created from

>    Amazon sales orders are **not** synchronized with TaxCloud since Amazon already includes them in
>    its own tax report to TaxCloud.

Reviewer 2: ```suggestion
   Amazon sales orders are **not** synchronized with TaxCloud, since Amazon already includes them in
   its own tax report to TaxCloud.
```

Register payments
-----------------

As customers pay Amazon as an intermediary, creating a dedicated *Bank* journal with a dedicated

> *Bank and Cash* intermediary account is recommended.

Reviewer: Im not sure to understand this and what we are trying to recommend and it does not sound logical with "Amazon journal" used after in the text

Author: I will clarify by suggesting a name for this journal.
Christophe said the flow was ok (content taken from the current documentation).

In addition, as Amazon makes a single monthly payment, selecting all the invoices linked to a single
payment is necessary when registering payments. Use the dedicated Amazon :guilabel:`Journal`

> you created and select :guilabel:`Batch Deposit` as the :guilabel:`Payment Method`. Then, select all the
> payments generated and click :menuselection:`Actions --> Create batch payment --> Validate`.

Reviewer 2: ```suggestion
that was created, and select :guilabel:`Batch Deposit` as the :guilabel:`Payment Method`. Then,
select all the payments generated, and click :menuselection:`Actions --> Create batch payment -->
Validate`.
```

.. tip::

>    You can do the same with vendor bills from Amazon dedicated to commissions. When you receive the
>    balance on your bank account at the end of the month and record your bank statements, credit the
>    Amazon intermediary account by the amount received.

Reviewer 2: ```suggestion
   The same can be done with vendor bills from Amazon dedicated to commissions. When the balance is
   received on the bank account at the end of the month, and the bank statements are recorded,
   credit the Amazon intermediary account by the amount received.
```
